docs/api/scheduler.md

```diff
@@ -375,6 +377,9 @@ In the example below there are three allocations belonging to two applications.
                     "kubernetes.io/meta/namespace": "default",
                     "kubernetes.io/meta/podName": "task0"
                 },
+                "requestTime":1648754034098912461,
```

Comment:
Let's add a space after these to be consistent with the rest of the file. There's several instances of this.